docs/source/models.rst

```diff
@@ -2738,23 +2738,52 @@ In order to use the bitrate transposition and conversion of the audio bytes data
 Installing this package directly from pypi (`pip install ffmpeg`) does not install the underlying `c` dll's that are required to make `ffmpeg` function.
 Please consult with the documentation at `the ffmpeg website <https://ffmpeg.org/download.html>`_ for guidance on your given operating system.
 
-The Audio Pipeline types, when loaded as a :ref:`python_function (pyfunc) model flavor <pyfunc-model-flavor>` have two input types available:
+The Audio Pipeline types, when loaded as a :ref:`python_function (pyfunc) model flavor <pyfunc-model-flavor>` have three input types available:
 
-* `bytes`
+* ``str``
+
+The string input type is meant for blob references (uri locations) that are accessible to the instance of the ``pyfunc`` model.
+This input mode is useful when doing large batch processing of audio inference in Spark due to the inherent limitations of handling large ``bytes``
+data in ``Spark`` ``DataFrames``. Ensure that you have ``ffmpeg`` installed in the environment that the ``pyfunc`` model is running in order
+to use ``str`` input uri-based inference. If this package is not properly installed (both from ``pypi`` and from the ``ffmpeg`` binaries), an Exception
+will be thrown at inference time.
+
+.. warning:: If using a uri (`str`) as an input type for a `pyfunc` model, you *must* specify a custom model signature when logging or saving the model.
+    The default signature value of ``bytes`` will, in `MLflow Model serving`, force the conversion of the string to ``bytes``, which will cause an Exception
```

Comment:
Q: Can we avoid automatically converting string input to `bytes` but raising error instead ?

Reply:
Good idea. I'll see if I can make this work!